packages/plugins/plugin-space/src/translations.ts

```diff
@@ -47,6 +47,7 @@ export default [
         'delete object label': 'Delete item',
         'collection deleted label': 'Collection deleted',
         'object deleted label': 'Item deleted',
+        'objects deleted label': 'Items deleted',
```

Comment:
see https://github.com/dxos/dxos/blob/da7e0b3e001c14e9c78cb47add1c95ab01a5c651/packages/plugins/plugin-space/src/translations.ts#L70-L72

Reply:
Hmm, none of the `{{count}}` translations I could find seem to have usages. How do I supply the params?

Reply:
https://www.i18next.com/translation-function/plurals

Reply:
https://github.com/dxos/dxos/blob/a597fa5190f0fe057687eecb8d5bfea610eda224/packages/plugins/plugin-space/src/SpacePlugin.tsx#L1347-L1359

Hmmm.

Reply:
Ah yeah that would be problematic, just TODO I guess